generators/app/templates/src/index.js

```diff
@@ -86,6 +86,6 @@ export const <%= toolNameCamel %> = {
   init: initComponent()
 };
 
-// this line connects the html element in idex.html with the javascript
+// this line connects the html element in index.html with the javascript
 // defined above.
```

Comment:
Can you have the “defined above.” also in the same single line comment?

Reply:
@sarthak-sehgal Done! :)